pkgs/development/libraries/fbthrift/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub, cmake, bison, boost, libevent, double-conversion
+, fizz, flex, fmt, folly, glog, google-gflags, libiberty, openssl, rsocket-cpp
+, wangle, zlib, zstd }:
+
+stdenv.mkDerivation rec {
+  name = "fbthrift-${version}";
+  version = "2019.06.10.00";
+
+  src = fetchFromGitHub {
+    owner = "facebook";
+    repo = "fbthrift";
+    rev = "v${version}";
+    sha256 = "1fx7awkmsqiy92nsxd78zfpkfvga8xwdhza87kml17v0zspf3b2w";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [
+    bison
```

Comment:
Are all of these really linked against? Flex and bison are usually only used during build.